MaybeGame/GameMaster.cs

```diff
@@ -1,10 +1,63 @@
-﻿namespace MaybeGame
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MaybeGame
 {
-    partial class Program
+    partial class MainProgram
```

Comment:
???????

Reply:
Renamed Program.cs To MainProgram.cs and it added it here so that needed to be changed

Reply:
Program.cs always stays Program.cs, don't rename it. Get rid of  **partial**